### ISSUE TITLE
[Question] Are there any confirmed ways to get demon cocks? (besides the Omnibus Overseer)

### ISSUE THREAD

**Issue by SargMjr24** [Kitteh6660/Corruption-of-Champions-Mod#1432]
### Overview
Looking for any ways to obtain infernal dicks.

**Comment by Stadler76** [comment]
Now, that I see it: Incubus Draft doesn't turn your cocks to demon cocks. And `Mutations.growDemonCock()` just grows a cock, but no demon cock.
The latter could be easily fixed:
```diff
diff --git a/classes/classes/Items/Mutations.as b/classes/classes/Items/Mutations.as
index 002d8a9d9..bdf1bb2d4 100644
--- a/classes/classes/Items/Mutations.as
+++ b/classes/classes/Items/Mutations.as
@@ -997,11 +997,7 @@ package classes.Items
 		{
 			temp = 0;
 			while (growCocks > 0) {
-				player.createCock();
-				//trace("COCK LENGTH: " + player.cocks[length - 1].cockLength);
-				player.cocks[player.cocks.length - 1].cockLength = rand(3) + 4;
-				player.cocks[player.cocks.length - 1].cockThickness = .75;
-				//trace("COCK LENGTH: " + player.cocks[length - 1].cockLength);
+				player.createCock(rand(3) + 4, .75, CockTypesEnum.DEMON);
 				growCocks--;
 				temp++;
 			}
```

The former might need some de-spaghettifying the code first.

**Comment by SargMjr24** [comment]
What do I do with the code?

**Comment by Stadler76** [comment]
Accidentally closed? Reopening.

**Comment by SargMjr24** [comment]
Oops. Didn't mean to close. What do I do with that code?

**Comment by SargMjr24** [comment]
?

**Comment by SargMjr24** [comment]
I actually just ended up using the save editor lol

**Comment by Stadler76** [comment]
> Oops. Didn't mean to close. What do I do with that code?

Its a diff. It shows, which file has to be edited and which lines have to be removed and which have to be added. After that, you have to compile the game. A guide for this can be found [in the forum](https://forum.fenoxo.com/threads/guide-how-to-make-a-mod-for-corruption-of-champions.3283/).

> I actually just ended up using the save editor lol

For your question: Currently large canine pepper changes your cock to a demon cock if you have a single cock (untested) and the Exgartuan is a demon cock.

PS: I might implement, so that incubus draft turns cocks to demon cocks and newly grown cocks might become demon cocks, too. Hence, why I'll reopen this so I don't forget it.

**Comment by SargMjr24** [comment]
I tested it and can confirm that Large Canine Peppers do NOT, I repeat, do NOT turn your cock into a demon cock if you have a single cock.

**Comment by Stadler76** [comment]
Yeah, looking at the code: If you already have a demon cock, the canine (each of them it seems) pepper won't change it to a dog cock. Well, that throws that option off the table, too.

**Comment by SargMjr24** [comment]
That sounds good to me. I mean, it makes sense that after spending so much time in Mareth that that much corruption would simply make your cocks demonic at that point.

P.S. Is Omnibi Juice in the game yet? If so, how do I get it?

**Comment by SargMjr24** [comment]
Is Omnibi Juice in the game yet? If so, how do I get it?

P.S. How is the Hellcat stuff going?

**Comment by SargMjr24** [comment]
Or is it being abandoned?

**Comment by SargMjr24** [comment]
...

**Comment by Stadler76** [comment]
Procrastination stroke again and now CoC UEE is in a feature freeze. You might wanna ask the original author of the gdoc if its implemented in a different CoC mod.
